### DISCUSSION TITLE
[Pimcore X] Override wysiwyg config

### DISCUSSION THREAD

**Discussion by valentin-** [pimcore/pimcore#8960] [Q&A / Support]
Hi, 

I'm trying to override the global configuration for the wysiwyg editor as described in https://pimcore.com/docs/pimcore/master/Development_Documentation/Documents/Editables/WYSIWYG.html#page_Global-Configuration.

The JS file is added using an event listener (https://pimcore.com/docs/pimcore/master/Development_Documentation/Extending_Pimcore/Bundle_Developers_Guide/Plugin_Backend_UI.html).

I've tried to update the config with plain JS and adding the code as a plugin using `postOpenDocument` and `pimcoreReady` event, but `pimcore.document.editables.wysiwyg` is not available.

Aren't the docs up to date or did I miss something? There is also no key for `pimcore.document.editables.wysiwyg` in the browser console.

Thanks!

**Comment by valentin-** [comment]
@dvesh3 could you please help me and provide an example? Thanks!

**Reply by dvesh3** [reply]
Either register global configuration via event `BundleManagerEvents::EDITMODE_JS_PATHS` https://pimcore.com/docs/pimcore/current/Development_Documentation/Documents/Editables/WYSIWYG.html#page_Registering-global-configuration-via-events 

or when using bundle, just return js file in `getEditmodeJsPaths()` with config described here https://pimcore.com/docs/pimcore/current/Development_Documentation/Documents/Editables/WYSIWYG.html#page_Global-Configuration
![image](https://user-images.githubusercontent.com/5137917/125750181-6004e102-7a54-4ac0-83c6-f6d2ea4aba48.png)

In js file, initialize defaultEditorConfig:
```
pimcore.document.editables.wysiwyg.defaultEditorConfig = pimcore.document.editables.wysiwyg.defaultEditorConfig || {};
pimcore.document.editables.wysiwyg.defaultEditorConfig.uiColor = '#AADC6E';
```

**Reply by valentin-** [reply]
I found the issue. I had a service definition to load the js files. Now I've implemented `EventSubscriberInterface` and `getSubscribedEvents()` and removed it from services.yml.

Thanks!
